lmdeploy/cli/utils.py

```diff
@@ -222,18 +245,6 @@ def repetition_penalty(parser):
                                    default=1.0,
                                    help='Parameter to penalize repetition')
 
-    @staticmethod
-    def cap(parser):
-        """Add argument cap to parser."""
-
-        return parser.add_argument(
-            '--cap',
-            type=str,
-            default='chat',
-            choices=['completion', 'infilling', 'chat', 'python'],
-            help='The capability of a model. '
```

Comment:
May update https://github.com/InternLM/lmdeploy/blob/main/autotest/utils/run_client_chat.py#L32

Reply:
@zhulinJulia24 I remove `--cap` from the CLI chat and use `--chat-template <a json file>` instead.
Could you help fixing the test cases?

Reply:
> @zhulinJulia24 I remove `--cap` from the CLI chat and use `--chat-template <a json file>` instead. Could you help fix the test cases?

already fixed in pr https://github.com/InternLM/lmdeploy/pull/2183